test/e2e/common/secrets.go

```diff
@@ -180,14 +201,19 @@ func secretForTest(namespace, name string) *api.Secret {
 	}
 }
 
-func doSecretE2EWithoutMapping(f *framework.Framework, defaultMode *int32) {
+func doSecretE2EWithoutMapping(f *framework.Framework, defaultMode *int32, secret2 *api.Secret) {
```

Comment:
Yep. As long as we have the variable name as `secret2` in the `It() block`, I don't mind changing the parameter of `doSecretE2EWithMapping()`.
I'll make the change.